docs/Relays.md

```diff
@@ -0,0 +1,27 @@
+AutoFixture is focused on creating instances of concrete types. For this reason it doesn't have any support for non-concrete types like interfaces and abstract classes.
+
+Most of the time, AutoFixture needs to resort to other libraries, like Moq, to create objects implementing an interface or representing a dummy specialization of an abstract class.
```

Comment:
Is it AutoFixture that resort to other libraries or the test author?

Reply:
Well, both in a sense. Test developer can use external libraries to generate mocks, but also could instruct AutoFixture to do so when dealing with interfaces.